highfidelity/hifi
# Case 5783: Restarting Interface allows you to have Avatar Scale Too Big

Fix for case 5783: Restarting Interface allows you to have Avatar Scale Too Big
https://highfidelity.fogbugz.com/f/cases/5783

**=================
Bug details
=================**
In Settings > Avatar, you can set your Avatar scale. If you pick a number > 3 (say, 14.0), the scale is capped at 3. However, there is a bug that lets you get around the maximum:
1. Go to Settings > Avatar > Avatar Tunning grid
2. Change Avatar Scale to 14.0 (any number greater than 3)
3. Click Save Changes. 
4. Restart Interface. Notice that you are much bigger than a 3.0 scale.
Fix the bug so that even if you restart interface, your Avatar scale is capped at 3.0.

**=================
How To Test
=================**

1. Open Interface window
2. Go to Settings > Avatar
3. Change Avatar Scale to any number greater than 3. [For example 15]
4. Click Save Changes. 
5. Close the interface
6. Restart Interface

**=================
Expected Result 
=================**
Avatar scale will be 3 when you restart the interface. Not the number you have set [For example 15] before closing the interface.
If you have set any number less than 3 [For example 2]. Then after restart there will be no change in Avatar scale, it remains the same. [For example 2]
